install.sh

```diff
@@ -59,7 +59,7 @@ if [ -z "${QUICKSPRINT_SKIP_IMAGE_INSTALL:-}" ]; then
 fi
 
 TARBALL=""
-case "$OS/$ARCH" in
+case "${OS}/${ARCH}" in
```

Comment:
I think this is okay as it should still match the MINGW64_NT* case due to the wildcard ending there.

Thanks for working on this and getting the updated docker.

Reply:
This particular change doesn't change any functionality, it's just a more restrictive use of the variables.